component/stream/read.go

```diff
@@ -201,6 +204,16 @@ func (r *ReadCache) WriteFile(options internal.WriteFileOptions) (int, error) {
 	return 0, syscall.ENOTSUP
 }
 
+func (r *ReadCache) FlushFile(options internal.FlushFileOptions) error {
+	// log.Trace("Stream::FlushFile : name=%s, handle=%d", options.Handle.Path, options.Handle.ID)
+	err := r.NextComponent().FlushFile(options)
```

Comment:
In ReadCache we can skip calling flush for next component, as file-system is read-only so there will not be any change of contents. We can just return back nil here.